robot/src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -75,6 +92,14 @@ public Drive(DreadbotMotor fl, DreadbotMotor fr, DreadbotMotor bl, DreadbotMotor
         diffDrive = new DifferentialDrive(leftMotors, rightMotors);
         slewRate = new SlewRateLimiter(DriveConstants.SLEW_RATE_LIMIT, -DriveConstants.SLEW_RATE_LIMIT, 0.2);
     }
+    @Override
+    public void periodic() {
+        odometry.update(
+            gryo.getRotation2d(),
+            (frontLeftMotor.getEncoder().getPosition() / AutonomousConstants.ROTATIONS_PER_METER),
+            (frontRightMotor.getEncoder().getPosition() / AutonomousConstants.ROTATIONS_PER_METER)
```

Comment:
Here we use the individual motor encoder, and the value needs be correctly inverted

Setting the inverted on the motors would work here, setting it on the motorGroups would not.